packages/edit-site/src/components/editor/global-styles-provider.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext, useContext, useEffect } from '@wordpress/element';
+import { useEntityProp } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import { getGlobalStyles } from './global-styles-renderer';
+
+const GlobalStylesContext = createContext( {
+	/* eslint-disable no-unused-vars */
+	getProperty: ( context, family, name ) => {},
```

Comment:
We are assuming always two levels, family-> name would it make sense to have a path and use arbitrary levels like lodash?

Reply:
In working to acomodate the gradients panel, this notion of paths ala lodash came in handy, so I just did that!